.travis.yml

```diff
@@ -0,0 +1,10 @@
+sudo: false
+language: node_js
+node_js:
+  - "4"
+  - "5"
+  - "node"
```

Comment:
"node" = current, which is 6, but we can future proof it a bit by adding 6 explicitly. I'll add it.